sklearn/preprocessing/data.py

```diff
@@ -2209,6 +2209,17 @@ def __init__(self, n_quantiles=1000, output_distribution='uniform',
         self.random_state = random_state
         self.copy = copy
 
+    @staticmethod
+    def _nanpercentile_force_finite(a, q):
```

Comment:
`a` and `q` are bad variable names. Please use something more descriptive e.g.: `column_data` and `percentiles`.

Reply:
it was to mimic the numpy function:
https://docs.scipy.org/doc/numpy-dev/reference/generated/numpy.nanpercentile.html

Do you still want to change?